frontend-react/src/components/header/ReportStreamNavbar.tsx

```diff
@@ -24,6 +25,14 @@ import styles from "./ReportStreamNavbar.module.scss";
 
 const { IS_PREVIEW, CLIENT_ENV, APP_ENV } = config;
 
+const primaryLinkClasses = (isActive: boolean) => {
```

Comment:
re-usable tool for checking if a link should be considered in an `active` state.